pyproject.toml

```diff
@@ -0,0 +1,28 @@
+[build-system]
+requires = ["poetry-core"]
+build-backend = "poetry.core.masonry.api"
+
+[tool.poetry]
+name = "cps"
+version = "0.dev"
+description = ""
+authors = ["The CPS Project <@>"]
+readme = "README.md"
+packages = [
+  { include = "cps.py", from = "_extensions" },
+  { include = "autosectionlabel.py", from = "_extensions" }
```

Comment:
I've half a mind to leave this as `_ext`, though I suspect "brevity" is the main motivation for that, and thus I feel I shouldn't override you if you feel strongly. WDYT?

Reply:
I think we should make it clear they're extensions and not maybe any other word that could be confused with it. `_extended` for example.